<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointCloudControl</class>
 <widget class="QWidget" name="PointCloudControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="pointSizeLayout">
     <item>
      <widget class="QLabel" name="pointSizeTextLabel">
       <property name="text">
        <string>Point size:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="pointSizeSpinBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="smoothPointsCheckBox">
       <property name="text">
        <string>Smooth</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ColorChooser" name="colorChooser" native="true"/>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="showLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="showPointsCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show points</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="browseLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="logEdit">
       <property name="text">
        <string>/home/jmaye/ETHZ/svn/curb-detection/data</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="logBrowseButton">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="logTextLabel">
       <property name="text">
        <string>Log file:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="testAction">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorChooser</class>
   <extends>QWidget</extends>
   <header>ColorChooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>logBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>PointCloudControl</receiver>
   <slot>logBrowseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointSizeSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PointCloudControl</receiver>
   <slot>pointSizeChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smoothPointsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PointCloudControl</receiver>
   <slot>smoothPointsToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showPointsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PointCloudControl</receiver>
   <slot>showPointsToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>logBrowseClicked()</slot>
  <slot>pointSizeChanged(double)</slot>
  <slot>smoothPointsToggled(bool)</slot>
  <slot>showPointsToggled(bool)</slot>
 </slots>
</ui>
